config/default.yml

```
@@ -3142,13 +3142,14 @@ Style/AccessModifierDeclarations:
   Description: 'Checks style of how access modifiers are used.'
   Enabled: true
   VersionAdded: '0.57'
-  VersionChanged: '0.81'
+  VersionChanged: '<<next>>'
```

Since the default values of existing configuration options are not changing, this modification is unnecessary.

```suggestion
  VersionChanged: '0.81'
```

---

Hmm, I thought we were supposed to update it when we configuration options were added as well. (not only on default behavior changes)

---

Ah, "`new config`" is indeed mentioned below:
https://github.com/rubocop/rubocop/issues/5978

OTOH, my understanding of `VersionChanged` is that it applies to cases where the default configuration changes the existing detection method for users (e.g., changing of value of `EnforcedStyle`). I considered the addition of a configuration to prevent excessive detections, like in this case, to be closer to a bug fix. That said, in practice, this metadata is not yet being utilized effectively, so either approach currently poses no significant issue.

Regarding the quoted issue content, it may be information that exists only in the issue. I will verify it later and document it if necessary.

---

The content based on issue #5978 has been documented in PR #13660.

---

Given the above / #13660, I think updating `VersionChanged` here is valid. @koic are you ok with it now?